main/templatetags/cdn.py

```diff
@@ -11,14 +11,14 @@ def jquery():
     version = '3.3.1'
     filename = 'jquery-%s.min.js' % version
     link = staticfiles_storage.url(filename)
-    return mark_safe('<script type="text/javascript" src="%s"></script>' % link)
+    return mark_safe('<script src="%s"></script>' % link)
 
 
 @register.simple_tag
 def jquery_tablesorter():
     version = '2.31.0'
     filename = 'jquery.tablesorter-%s.min.js' % version
     link = staticfiles_storage.url(filename)
-    return format_html('<script type="text/javascript" src="%s"></script>' % link)
+    return format_html('<script src="%s"></script>' % link)
```

Comment:
All these type="text/javascript" changes have nothing to do with responsive layout. They should be done in a separate commit.